Sources/Plasma/PubUtilLib/plAvatar/plAvBrainCritter.cpp

```diff
@@ -651,21 +652,16 @@ std::vector<unsigned long> plAvBrainCritter::IGetAgePlayerIDList() const
 {
     // make a list of non-local players
     std::vector<unsigned long> playerIDs;
-    std::map<unsigned long, bool> tempMap; // slightly hacky way to remove dups
     plNetClientMgr* nc = plNetClientMgr::GetInstance();
     for (size_t i = 0; i < nc->TransportMgr().GetNumMembers(); ++i)
     {
         plNetTransportMember* mbr = nc->TransportMgr().GetMember(i);
         unsigned long id = mbr->GetPlayerID();
-        if (tempMap.find(id) == tempMap.end())
-        {
-            playerIDs.push_back(id);
-            tempMap[id] = true;
-        }
+        playerIDs.push_back(id);
```

Comment:
This won't remove any duplicates from the TransportMgr's list, which the old code also did...  I suppose you could also just use a set and then convert it back to a vector (if necessary)